### PR TITLE
Adapt notebook to work with cinnabar 0.4

### PR DESCRIPTION
**Pull request by hannahbaumann**

### CODE REVIEW

**Comment by IAlibay** [comment]
@hannahbaumann could you add this folder to the CI command? We should try to have it run here too

File: plotting_rbfes_with_cinnabar/PlottingFreeEnergiesUsingCinnabar.ipynb [hunk 1]
```diff
@@ -360,66 +377,66 @@
...
    "source": [
     "dG_results = {}\n",
-    "nodes = fe.graph.nodes(data=True)\n",
+    "nodes = list(fe.to_legacy_graph().nodes.data())\n",
     "\n",
     "for key in range(len(nodes)):\n",
-    "    dG_results[nodes[key]['name']] = {\n",
-    "        'experimental_estimate': nodes[key]['exp_DG'],\n",
-    "        'experimental_error': nodes[key]['exp_dDG'],\n",
-    "        'calculated_estimate': nodes[key]['exp_DG'],\n",
```
Comment (IAlibay) [inline plotting_rbfes_with_cinnabar/PlottingFreeEnergiesUsingCinnabar.ipynb:421]: yikes, that was a bad typo 🙈 - Thanks for catching this!

File: plotting_rbfes_with_cinnabar/PlottingFreeEnergiesUsingCinnabar.ipynb [hunk 2]
```diff
@@ -360,66 +377,66 @@
...
       "                'experimental_estimate': -11.74},\n",
-      " 'lig_jmc_27': {'calculated_error': 0.17,\n",
-      "                'calculated_estimate': -11.31,\n",
+      " 'lig_jmc_27': {'calculated_error': 0.11407892005098923,\n",
+      "                'calculated_estimate': -1.288999999999997,\n",
       "                'experimental_error': 0.17,\n",
       "                'experimental_estimate': -11.31},\n",
-      " 'lig_jmc_28': {'calculated_error': 0.18,\n",
-      "                'calculated_estimate': -11.01,\n",
+      " 'lig_jmc_28': {'calculated_error': 0.07933473388119473,\n",
```
Comment (IAlibay) [inline plotting_rbfes_with_cinnabar/PlottingFreeEnergiesUsingCinnabar.ipynb:423]: Could you do a 2d.p. truncation on the results writing so calculated & experimental are consistent? (lines 452-453 & 463-464)

File: plotting_rbfes_with_cinnabar/PlottingFreeEnergiesUsingCinnabar.ipynb [hunk 3]
```diff
@@ -447,6 +464,14 @@
     "            dG_results[ligand]['experimental_error'],\n",
     "        ])"
    ]
+  },
+  {
+   "cell_type": "code",
```
Comment (IAlibay) [inline plotting_rbfes_with_cinnabar/PlottingFreeEnergiesUsingCinnabar.ipynb:469]: Could you delete this cell please?

File: plotting_rbfes_with_cinnabar/PlottingFreeEnergiesUsingCinnabar.ipynb [hunk 4]
```diff
@@ -22,7 +22,7 @@
     "from pprint import pprint\n",
     "import cinnabar\n",
     "from cinnabar import plotting as cinnabar_plotting\n",
-    "from cinnabar.wrangle import FEMap"
+    "from cinnabar import femap, stats"
```
Comment (IAlibay) [inline plotting_rbfes_with_cinnabar/PlottingFreeEnergiesUsingCinnabar.ipynb:25]: I tried running this locally with a fresh install of cinnabar:

```
# packages in environment at /home/ialibay/software/mambaforge/install/envs/openfe:
#
# Name                    Version                   Build  Channel
cinnabar                  0.3.0              pyhd8ed1ab_0    conda-forge
```

and I'm getting an import error on `femap`

**Review by IAlibay — CHANGES_REQUESTED** [review]
Thanks @hannahbaumann - couple of small things + CI

I wasn't able to run this locally on a fresh cinnabar install though :/

**Comment by richardjgowers** [comment]
@IAlibay I suspect it might need a cinnabar release, I'll do it this morning

**Review by IAlibay — APPROVED** [review]
All works fine locally, thanks @hannahbaumann !

**Pull request merged**